Dockerfile.release

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:22.04
+
+# Setup mount and plugin directories needed for the plugin
+RUN mkdir -p /run/docker/plugins /mnt/state /mnt/volumes
+
+# Install dependencies
+RUN apt update && \
+    apt install -y git nfs-common && \
+    rm -rf /var/lib/apt/lists/*
+
+# Setup a default git user/email
+RUN git config --global user.name "gatling" && \
+    git config --global user.email gatling@xethub.com
```

Comment:
TODO: we could probably use docker build args to set these or have them set via options on volume creation.